dist/components/TopBar/TopBar.css

```diff
@@ -0,0 +1,45 @@
+@import '../../css/base.css';
+
+.top-bar {
+  width: 100%;
+  background-color: var(--g3-color__base-blue);
+  display: flex;
+  align-items: center;
+}
+
+.top-bar__header {
+  background-color: var(--g3-color__base-blue);
+  overflow: hidden;
+  vertical-align: middle;
+  margin: auto;
+  width: 1216px;
```

Comment:
I think we can create a width variable like `--g3-desktop-width` in base.css. And also I found in some pages like `DataExplorer` the width is `1220px`

Reply:
I think that all the hardcoded widths should just be removed in general so that the page is more responsive - do you think we should make a separate ticket?

Reply:
Yes~ Thanks!